src/main/webapp/util/Validator.js

```diff
@@ -41,10 +72,6 @@ class Validator {
 
     // Validate the string length of field
     StringValidator(field, label, validator, data) {
-        const { error } = parseStringValidator(validator.minLength, validator.maxLength);
-        if (error) {
-            return { errorField: field, errorMsg: error };
-        }
```

Comment:
We removed this because we have the same code below (which checks for the minLength and maxLength)?

Reply:
Actually, this code was unused. It basically validates the min and max length provided in the global config file when loading the TA, and this validation is already available in the ucc-gen.